letta/orm/organization.py

```diff
@@ -0,0 +1,35 @@
+from typing import TYPE_CHECKING
+
+from sqlalchemy.exc import NoResultFound
+from sqlalchemy.orm import Mapped, mapped_column
+
+from letta.orm.sqlalchemy_base import SqlalchemyBase
+from letta.schemas.organization import Organization as PydanticOrganization
+
+if TYPE_CHECKING:
+    from sqlalchemy.orm import Session
+
+
+class Organization(SqlalchemyBase):
```

Comment:
Why does this not use the `OrganizationMixin`? Also, do we need the `OrganizationMixin` if we only have one ORM model that needs it?

Reply:
removed OrganizationMixin for now from this PR, OrganizationMixin is meant to be used for other models that require a tie back to an organization